MI-Xib_cell/Controllers/ViewController.swift

```diff
@@ -42,17 +42,17 @@ extension ViewController: UITableViewDelegate, UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: CustomTableViewCell.reuseId, for: indexPath) as? CustomTableViewCell else {return .init()}
-        
-        cell.configure(labelList: labelData)
+        let data = labelData[indexPath.row]
+        cell.roleLabel.text = data.role
+        cell.nameLabel.text = data.participantName
+//        cell.configure(labelList: labelData)
```

Comment:
at cell at method configure set this properties

func configure(data: LabelData){
roleLabel.text = data.role
}